### PR TITLE
Slovene translations

### PR DESCRIPTION
**Pull request by sysadminmatmoz**
No code changes or additions, just slovene translations added to several modules

### CODE REVIEW

File: mass_editing/i18n/sl.po [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+msgstr "Opozorilo"
+
+#. module: mass_editing
+#: help:mass.object,ref_ir_act_window:0
+msgid ""
+"Sidebar action to make this template available on records                     "
+"                             of the related document model"
+msgstr ""
+"Dejanje stranske vrstice, ki omogoči razpoložljivost te predloge na zapisih   "
+"                  "
```
Comment (pedrobaeza) [inline mass_editing/i18n/sl.po]: Remove this extra line and spaces

Comment (pedrobaeza) [inline mass_editing/i18n/sl.po]: FYI, this is just a side effect of bad code on the modules. Better to fix it on translations.

**Comment by pedrobaeza** [comment]
Good to meet people from Slovenia! A little remark only from technical side, and nothing to say about slovenian itself.

**Comment by sysadminmatmoz** [comment]
Done :)

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1: LGTM (just reviewed it matches the last pot template)

**Comment by pedrobaeza** [comment]
I proceed with the merge due the triviality of the PR.

**Pull request merged**